oefen_api/.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Flask",
+            "type": "python",
+            "request": "launch",
+            "module": "flask",
+            "env": {
+                "FLASK_APP": "text-api.py",
+                "FLASK_ENV": "development",
+                "FLASK_DEBUG": "0"
+            },
+            "args": [
+                "run",
+                "--no-debugger",
+                "--no-reload"
+            ],
+            "jinja": true
+        }
+    ]
+}
```

Comment:
Het is een 'best practice' om altijd een `newline` character als laatste in elke file te zetten. Zie https://stackoverflow.com/a/5813359

je kan dit automatisch door je editor laten doen b.v met een `.editorconfig`en de bijbehorende editor plugin. (voor vscode https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)